src/quantum/widget.jsx

```diff
@@ -4,102 +4,115 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const enrich = (text, key = 'none') => (typeof text === 'string'
-  ? (
+const enrich = (text, key = 'none') =>
+  typeof text === 'string' ? (
     <span
       key={`enriched-${key}`}
+      // eslint-disable-next-line react/no-danger
```

Comment:
I'd find a way to replace the usage of `dangerouslySetInnerHTML` instead of disabling, or file a bug if there isn't one already to replace all uses of it.